.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Question
+about: I have a question
```

Comment:
Do we want to direct people at all about what questions this should be used for? What prompted you to add this? Product questions on docs?

Reply:
That and I've also seen issues that are pseudo-questions that I think if we had a question template that we'd get more straightforward questions instead of bug-as-questions. If we don't get much usage then we can ditch it if my hypothesis is wrong. :)